### DISCUSSION TITLE
Welcome to First1KGreek Discussions!

### DISCUSSION THREAD

**Discussion by gregorycrane** [OpenGreekAndLatin/First1KGreek#2755] [Announcements]
# Thoughts on updating OGL (Firs1K and Perseus)
  This is to collect random thoughts about OGL on which we may or may not act.

## Author Names

We have traditionally followed print models in identifying authors, e.g., A. M. Harmon and K. Kilburn as editors/translators of Lucian. We should work towards providing 

* Expanded initial (full names)
* Birth and death dates
* Links to one or more authority lists.

## Issues in the header

Time moves on and this will soon look like "New Variorum Shakespeare" volumes that are more than 100 years old.
```
<p>Text encoded in accordance with the latest EpiDoc standards (January 2014) </p>
```
Replace with:
```
<p>Text encoded in accordance with the latest EpiDoc standards (as of January 2014) </p>
```
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lcerrato** [comment]
Counterpoint is that the header note on standards (an OGL work product) is a good hint that the headers need review. The change log and file history will tell you when the EpiDoc is reviewed anyhow. File history is more important and reliable, and if the changes are logged, the date will be there. No need to duplicate.

Expanded author info is what the catalog widget should provide, especially dates which would tend to clutter display. Not sure why a reader wants to see info that within the reading environment if there is a catalog widget/link or windowpane? Seems like there is more important (now hidden) edition info to prioritize than when a translator died? And we don’t generally re-enter data that something like the catalog or WorldCat can provide for us. That’s extra typing and data entry.

**Reply by AlisonBabeu** [reply]
I have to agree here, especially since the point (at least up until now), for the cts_.xml metadata has been to give basic citation information. Perhaps moving forwards (if we ever have a catalog widget), we could have a link from within the XML file to a separate catalog record (assuming one exists) in a widget,or to WorldCat in the way that we link out to the page level of the work.

**Comment by lcerrato** [comment]
`<p>Text encoded in accordance with the latest EpiDoc standards (January 2014)</p>`

Seems exclusive to OGL
DDD_XML
First1kGreek
patrologia_latina-dev

It is not part of the current Lace export header or the Perseus header recommendations. (No date is used in the current header recommendations.)

**Comment by abartov** [comment]
I warmly recommend using [Wikidata](https://wikidata.org) as a source for authority IDs, as it will _ipso facto_ get us many more identities; Wikidata is a nexus for lots of IDs, even more comprehensive than [VIAF](https://viaf.org).

Wikidata also has some tooling around reconciliation of names and authority IDs, chief among them [the Mix 'n' Match tool](https://mix-n-match.toolforge.org/#/).
